packages/fluentui/react-northstar/src/components/Breadcrumb/breadcrumbContext.ts

```diff
@@ -0,0 +1,12 @@
+import * as React from 'react';
+import { SizeValue } from '../../utils';
+
+export type BreadcrumbContextValue = {
+  size: SizeValue;
```

Comment:
```suggestion
  size: Exclude<SizeValue, 'larger' | 'largest'>;
```

We can drop a type there and reuse in other components